package.json

```diff
@@ -12,6 +12,10 @@
   "author": "CHEN Xian-an <xianan.chen@gmail.com>",
   "license": "MIT",
   "devDependencies": {
-    "bs-platform": "^1.7.3"
+    "bs-platform": "^2.2.3"
+  },
+  "dependencies": {
+    "reason-cli": "^3.1.0-darwin",
+    "refmt": "^1.13.7-1"
```

Comment:
`reason-cli` is editor tooling and should be globally installed, not a project dependency. `refmt` isn't something you should need at all, I think.